tempel.el

```diff
@@ -295,7 +295,7 @@ If OV is alive, move it."
 NAME is the optional field name.
 INIT is the optional initial input.
 Return the added field."
-  (let ((ov (make-overlay (point) (point)))
+  (let ((ov (make-overlay (point) (point) nil t))
```

Comment:
I think the current behavior was intentional. However if changing this will be beneficial overall, I am not opposed.

Reply:
Turns out this change was unnecessary. I've deleted this commit and change.

Reply:
Since this means the start position of a field/form overlay is never modified anymore, it also should be safe to force the invariant only for overlays after the moved overlay.